src/components/Search/Publications.tsx

```diff
@@ -75,8 +74,8 @@ const Publications: FC<Props> = ({ query }) => {
       loader={<InfiniteLoader />}
     >
       <Card className="divide-y-[1px] dark:divide-gray-700/80">
-        {publications?.map((post: LensterPublication, index: number) => (
-          <SinglePublication key={`${post?.id}_${index}`} publication={post} />
+        {publications?.map((post, index) => (
+          <SinglePublication key={`${post?.id}_${index}`} publication={post as LensterPublication} />
```

Comment:
Totally agree! I prefer that over casting too...

Reply:
Ouch... I just checked and I think is not possible on this case, since `publications` are of type `PublicationSearchResultItem` which is either `Comment` or `Post`, we need a `LensterPublication` type which it's different (a union between `Post`, `Mirror` and `Comment`) 😞